src/components/10-atoms/button/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@axa-ch/button",
-  "version": "0.0.0-beta.1",
+  "version": "1.0.0-alpha.1",
```

Comment:
we need to be consisten here everyone does its own versioning.. till today, every axa project used 0.0.0 as start version that on npm is officially as WIP and then tagged it with beta. I would vote for that naming convention but am ok also with 1.0.0-alpha which says 1.0.0 is almost ready.. but consistence is important so we need to define which format we will use